builds/volto-latest.yml

```diff
@@ -0,0 +1,86 @@
+steps:
+
+  # "kaniko" is a build runner whose sole job is to build
+  # container images optimised for use on k8s.
+  # destination is just where on our infrastructure we store
+  # these images.
+  # by default it looks for Dockerfile in the repo root and builds
+  # that though other locations can be specified.
+  # For why we build the image twice with different tags, see
+  # the cicd README.
+  - name: 'gcr.io/kaniko-project/executor:latest'
```

Comment:
can you add a space and include the comment for this step please

Reply:
Added space and the comment